site/src/pages/api/blocks/shared.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiResponse } from "next";
```

Comment:
We have parts of the codebase where a folder named `shared` is used to store resources re-used locally. Each file under `shared` is like a mini-library with a clearly defined scoped (= file name). Creating `shared.ts` means that _all_ shared resources will go to one file and it can become pretty messy. In this particular case, our resources are scattered between `src/pages/api/blocks/shared.ts` and `src/lib/api/blocks`. It’s not very clear why.

I suggest avoiding `shared.ts`, even if we have only few resources to put in there. It’s better to go for `shared/some-topic.ts` or at least stick with `src/lib/api/blocks` for consistency. The problem with the latter is that it’s too far from where the resources are actually used. It’s tolerable short-term though.

Reply:
https://github.com/blockprotocol/blockprotocol/pull/548/commits/02150ef4a20a4b506f30f99dd8682fb97b18bd59